src/react-components/button-link.tsx

```diff
@@ -9,7 +9,7 @@ import {
   buttonLinkStyle,
 } from "../theme/button-link.css"
 
-export type CustomHref = {
+export interface CustomHref {
```

Comment:
Why this change?

Reply:
eslint prefers interfaces over type definitions. since it changed the others since they were being used by other interfaces, this type was changed to reflect a consistent code-base. eslint is quite opinionated and it's not really necessarily about a technical reason, more about formatting in most cases.


https://typescript-eslint.io/rules/consistent-type-definitions/

Reply:
I'd prefer types then :D But consistency is good. Thanks for the explanation.